package.json

```diff
@@ -22,7 +22,10 @@
     "global:test": "cd $INIT_CWD && NODE_ENV=test jest",
     "global:typecheck": "cd $INIT_CWD && tsc --noEmit",
     "global:test-watch": "cd $INIT_CWD && NODE_ENV=test jest --watch",
-    "create-package": "yarn workspace ol-template plop package"
+    "create-package": "yarn workspace ol-template plop package",
+    "build-github-pages": "yarn workspace github-pages build",
+    "storybook": "yarn workspace mit-open storybook",
```

Comment:
That doesn't sound right, unless you're exec'ing into the container shell to start storybook? There's currently nothing firing up the storybook server inside the `watch` container, unless I'm missing something.

I don't see much value in running the development server inside Docker as it runs independently of any other service. If you do see a case for this, I'd suggest not using `watch`, instead adding a Docker Compose service specifically for it (single process/concern per container).

Reply:
> That doesn't sound right, unless you're exec'ing into the container shell to start storybook? 

Correct. For example, if I run `yarn storybook` directly on my host machine, then:
```
➜ yarn storybook
@storybook/cli v7.6.7

Error:
You installed esbuild for another platform than the one you're currently using.
This won't work because esbuild is written with native code and needs to
install a platform-specific binary executable.

Specifically the "@esbuild/linux-arm64" package is present but this platform
needs the "@esbuild/darwin-arm64" package instead. People often get into this
situation by installing esbuild on Windows or macOS and copying "node_modules"
into a Docker image that runs Linux, or by copying "node_modules" between
Windows and WSL environments.
```

If I run `yarn install` on host, then I suspect storybook would work, but our docker container running webpack won't work in that case (I'd have the darwin esbuild, whereas docker would want linux esbuild).